tests/integration/components/leaflet-map-test.js

```diff
@@ -3,6 +3,7 @@ import hbs from 'htmlbars-inline-precompile';
 import { assertionInjector, assertionCleanup } from '../../assertions';
 import LeafletMapComponent from 'ember-leaflet/components/leaflet-map';
 import locations from '../../helpers/locations';
+/* globals L */
```

Comment:
Sorry for being picky, but can you replace `L.point(500, 500)` with `[500, 500]` so we can remove this ugly `/* globals L */`?

I assume we will eventually shim leaflet so we can `import` it normally. So, the fewer places it is being used as global, the better.